src/components/Main.js

```diff
@@ -0,0 +1,22 @@
+import styled from "styled-components";
+
+const Main = styled.div`
+    display: grid;
+    grid-template-columns:30vw 1fr;
+    height: 100vh;
+    background: ${(props) =>
+        props.darkMode ? "var(--darkMain)" : "var(--lightMain)"};
+    color: ${(props) =>
+        props.darkMode ? "var(--darkText)" : "var(--lightText)"};
+    transform: ${(props) => (props.showInfo ? "translateX(-20vw)" : "none")};
+    filter: ${(props) => (props.showInfo ? "blur(3px)" : "none")};
```

Comment:
```suggestion
    transform: ${({ showInfo }) => (showInfo ? "translateX(-20vw)" : "none")};
    filter: ${({ showInfo }) => (showInfo ? "blur(3px)" : "none")};
```

You can use destructuring for a more cleaner code 😀